test/e2e/gitops_service_test.go

```diff
@@ -56,13 +60,14 @@ func TestGitOpsService(t *testing.T) {
 	t.Run("Validate ConsoleLink", validateConsoleLink)
 	t.Run("Validate ArgoCD Installation", validateArgoCDInstallation)
 	t.Run("Validate ArgoCD Metrics Configuration", validateArgoCDMetrics)
+	t.Run("Validate ArgoCD Installation", deployClusterScopedManifest)
 	t.Run("Validate tear down of ArgoCD Installation", tearDownArgoCD)
 }
 
 func validateGitOpsBackend(t *testing.T) {
 	framework.AddToFrameworkScheme(routev1.AddToScheme, &routev1.Route{})
 	framework.AddToFrameworkScheme(configv1.AddToScheme, &configv1.ClusterVersion{})
-	ctx := framework.NewTestCtx(t)
+	ctx := framework.NewContext(t)
```

Comment:
Are we planning to change this everywhere else in the testsuite in the same PR ?

Reply:
yes, its a minor change. So including it in the same pr.